go/tests.yaml

```diff
@@ -1,6 +1,7 @@
 vars:
   do_go_lint: true
   do_go_mod: true
+  do_go_format: true
```

Comment:
It seems to enable format for everything, what changes the current behavior that is isolated in a `if `case.